.github/CODEOWNERS

```
@@ -66,19 +66,28 @@ _posts/                      @kyle-patterson
 _data/                       @czubair @weshaggard
 android.md                   @czubair @vcolin7 @anuchandy
 android.yml                  @czubair @vcolin7 @anuchandy
+android-packages.csv         @czubair @vcolin7 @anuchandy
 c.md                         @czubair @rickwinter
 c.yml                        @czubair @rickwinter 
+c-packages.csv               @czubair @rickwinter
 cpp.md                       @czubair @rickwinter @vhvb1989 
 cpp.yml                      @czubair @rickwinter @vhvb1989
+cpp-packages.csv             @czubair @rickwinter @vhvb1989
 dotnet.md                    @czubair @JoshLove-msft
 dotnet.yml                   @czubair @JoshLove-msft
+dotnet-packages.csv          @czubair @JoshLove-msft
 ios.md                       @czubair @vcolin7 @tjprescott 
 ios.yml                      @czubair @vcolin7 @tjprescott
+ios-packages.csv             @czubair @vcolin7 @tjprescott
 java.md                      @czubair @g2vinay
 java.yml                     @czubair @g2vinay
+java-packages.csv            @czubair @g2vinay
```

I'm the release driver for Java now.
```suggestion
java.md                      @czubair @vcolin7
java.yml                     @czubair @vcolin7
java-packages.csv            @czubair @vcolin7
```